airflow/api_fastapi/core_api/routes/public/backfills.py

```diff
@@ -206,31 +206,20 @@ def create_backfill(
             reprocess_behavior=backfill_request.reprocess_behavior,
         )
         return BackfillResponse.model_validate(backfill_obj)
-    except AlreadyRunningBackfill:
-        raise HTTPException(
-            status_code=status.HTTP_409_CONFLICT,
-            detail=f"There is already a running backfill for dag {backfill_request.dag_id}",
-        )
-    except DagNoScheduleException:
-        raise HTTPException(
-            status_code=status.HTTP_409_CONFLICT,
-            detail=f"{backfill_request.dag_id} has no schedule",
-        )
-
-    except DagNotFound:
-        raise HTTPException(
-            status_code=status.HTTP_404_NOT_FOUND,
-            detail=f"Could not find dag {backfill_request.dag_id}",
-        )
+    except (
+        AlreadyRunningBackfill,
```

Comment:
`AlreadyRunningBackfill` needs to remain on it's own because it's a `409` conflict. 

`RequestValidationError` will return a 422.

Reply:
DagNotFound is a 404, same we don't want to return a validation error 422 on that.